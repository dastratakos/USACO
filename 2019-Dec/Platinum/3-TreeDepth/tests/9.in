49 843 343647949
