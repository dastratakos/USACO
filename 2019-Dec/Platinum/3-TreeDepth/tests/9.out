120757277 77103521 35835898 128975015 235812643 106808742 216028805 18726066 84134625 20268195 281451802 161450133 275259235 115134432 4449149 199813180 121762773 250026807 131614073 103004058 136607537 332402181 200780627 127537848 119189400 298169337 11950053 3478854 267676071 197536465 121537072 258964372 171640270 74969013 42079677 138780806 163875559 302261204 65824474 103039445 250374287 69138778 231138504 279914459 219508926 152007928 287619876 258731922 21182719
